VMAccess/vmaccess.py

```diff
@@ -79,6 +82,14 @@ def enable():
         if remove_user:
             _remove_user_account(remove_user, hutil)
         _set_user_account_pub_key(protect_settings, hutil)
+	if check_disk:
+            _fsck_check()
+            hutil.log("Successfully checked disk")
+            hutil.exit_if_enabled()
+        if repair_disk:
+            _fsck_repair()
+            hutil.log("Repaired and remounted disk")
+            hutil.exit_if_enabled()
```

Comment:
I think this line and the line 88 above are noops, since hutil.exit_if_enabled() would have exited on line 77 if this was not a new config according to the sequence number (https://github.com/Azure/azure-linux-extensions/blob/master/Utils/HandlerUtil.py#L252-L260).
So if you want to exit, then use hutil.do_exit...